src/components/issues/Card.tsx

```diff
@@ -69,6 +93,87 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
         }
     };
 
+    const onTaskRequestClick = () => {
+        setSelectedTab(REQUEST_TABS.CREATION_REQUEST);
+    };
+    const onTaskAssignmentClick = () => {
+        setSelectedTab(REQUEST_TABS.TASK_CREATION);
+    };
```

Comment:
- should we create a single function which accepts the tab we want to show? 
```js
const handleTaskRequestTabChange = (tab: REQUEST_TABS) => {
...
}
```

Reply:
Done.